Source/Scene/modelMaterialsCommon.js

```diff
@@ -349,6 +349,16 @@ define([
             vertexShader += 'attribute vec4 a_joint;\n';
             vertexShader += 'attribute vec4 a_weight;\n';
         }
+        
+        if (options.addBatchIdToGeneratedShaders)
+        {
+            techniqueAttributes.a_batchId = 'batchId';
+            techniqueParameters.batchId = {
+                semantic: 'BATCHID',
+                type: WebGLConstants.UNSIGNED_SHORT
```

Comment:
Quick question @lexaknyazev @pjcozzi 

Does a parameter's type equal the data type stored in the buffer or the type declared in the shader? If `batchId` is stored as unsigned shorts, should `type` be `5123` or `5126`.

```json
"technique": {
    "attributes": {
        "a_batchId": "batchId"
    },
    "parameters": {
        "batchId": {
            "semantic": "BATCHID",
            "type": 5123
        }
    }
}
```
Shader
```
attribute float a_batchId;
```